src/components/Properties/PropertyTitle.vue

```diff
@@ -23,11 +23,9 @@
 <template>
 	<h3 class="property__title property__row">
 		<div :class="icon" class="property__label property__title--icon" />
-		<span class="property__value property__title--right">
-			<div>{{ readableName }}</div>
-			<!-- display tooltip with hint if available -->
-			<div v-if="info" v-tooltip.auto="info" class="property__title--icon-details icon-details" />
```

Comment:
I still disagree with this.
We had to add this because lots of users are confused and vCards is complicated.
I don't want to remove it and I think it doesn't create any issues to leave it there

Reply:
Did you ever see something like this in another popular Contacts app?

Reply:
Let's ask the question differently, what would you say to a user that says they doesn't understand the difference between RELATED & RELATIONSHIP ? :)

Reply:
To that I would say that I fixed the wording via https://github.com/nextcloud/contacts/pull/1446/commits/715df614b33ada3b6ef8c358cec52ae2b9070e6b
→ to 'Relationship to you' and 'Related contacts' 

;)

Reply:
Is it more clear? Doesn't really seems to me :see_no_evil:

Reply:
Yes, it’s quite clear. One thing is to put in the "Relationship to you" andthe other thing is to put in "Related contacts". Not sure how much clearer it can be.

Reply:
Sure, your call!